tests/functional/object/rgw/test_rgw_http_endpoint.py

```
@@ -0,0 +1,298 @@
+        4. Validate unsecured RGW route is not present in route info
+        5. Try to curl unsecured address from local machine
+        6. Re-enable HTTP endpoint and verify route is recreated
+
+        """
+
+        # Step 1: Verify the default value present for disableHttp parameter
+        log.info("Step 1: Verifying default value of disableHttp parameter")
+        if not self.disable_http_exists:
+            log.info("disableHttp parameter does not exist (fresh cluster)")
+        else:
+            log.info(f"disableHttp parameter exists with value: {self.original_disable_http}")
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)